lib/tasks/provider.rake

```diff
@@ -19,4 +19,14 @@ namespace :provider do
   task :refresh_index => :environment do
     Provider.__elasticsearch__.refresh_index!
   end
+
```

Comment:
Again, why do we need aliases for the `providers` index? For zero downtime?

Reply:
this has been changed